src/components/PhotoAnalysis/PhotoAnalysis.tsx

```diff
@@ -50,10 +53,36 @@ const Analysis: React.FC = () => {
       .catch(err => setRekognitionResponse(JSON.stringify(err, null, 2)))
   }
 
+  const displayUploadButton = () => {
+    if (!isLoading && rekognitionResponse === "")
```

Comment:
We can't check for `rekognitionResponse` here, because it causes the upload button to hide after an image has been uploaded. Which is probably intended, but it has the side effect of requiring a user to refresh the page if they want to upload another image

Reply:
This was intentional. There is a another button at the top of the screen that allows the user to upload a new image and it will trigger the same functionality. We can talk about placement/design of that button but I don't think it affects the current task or this function